integrations/access/msteams/app.go

```diff
@@ -67,8 +78,7 @@ func NewApp(conf Config) (*App, error) {
 
 // Run starts the main job process
 func (a *App) Run(ctx context.Context) error {
-	log := logger.Get(ctx)
-	log.Info("Starting Teleport MS Teams Plugin")
+	a.log.InfoContext(ctx, "Starting Teleport MS Teams Plugin", "plugin_name", pluginName)
```

Comment:
```suggestion
	a.log.InfoContext(ctx, "Starting Teleport MS Teams Plugin")
```

Plugin name is always set in `NewApp()`